agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp

```diff
@@ -2241,8 +2241,7 @@ void monitor_thread::update_vaps_in_db()
 
         // if vap exist in HAL, update it in the local db.
         if (radio_vaps.find(vap_id) != radio_vaps.end()) {
-            auto iface_name = beerocks::utils::get_iface_string_from_iface_vap_ids(
```

Comment:
Do we have an issue for changing all `beerocks::utils::get_iface_string_from_iface_vap_ids` on the new method provided in this pull request?

Reply:
Moreover, I think this function should be removed. To prevent accident use of it.

Reply:
Yes, I created an issue to complete remaining work: PPM-325 remove get_iface_string_from_iface_vap_ids